core/modules/wbase/SendChannel.cc

```diff
@@ -188,6 +188,14 @@ bool SendChannel::kill() {
 }
 
 
+bool SendChannel::isDead() {
+    if (_dead) return true;
+    if (_ssiRequest == nullptr) return true;
```

Comment:
This question is for my own education - how it's possible that `_ssiRequest` was the `nullptr`? Is this because of a chance of constructing `SendChannel` using the default constructor? If that's the case then why one would use the default constructor at all? My understanding is that such a constructor would produce a completely useless object.

Reply:
It seems to show up in unti tests.